src/__tests__/FixedSizeGrid.js

```diff
@@ -475,6 +475,10 @@ describe('FixedSizeGrid', () => {
       rendered
         .getInstance()
         .scrollToItem({ columnIndex: 2, rowIndex: 2, align: 'auto' });
+      // Scroll down to row 10, without changing scrollLeft
+      rendered.getInstance().scrollToItem({ rowIndex: 10, align: 'auto' });
+      // Scroll left to column 0, without changing scrollTop
+      rendered.getInstance().scrollToItem({ columnIndex: 0, align: 'auto' });
```

Comment:
You aren't testing anything between these `scrollToItem` calls. That seems not great.

Reply:
Whoops. I misremembered how my own tests were written 😆 you're expecting on the mock calls below. 👍